### PR TITLE
add course page

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj [hunk 1]
```diff
@@ -9,6 +9,44 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Entities\MethodicDocument\MethodicDocumetsTableObject.cs" />
```
Comment (hryhoriivoloshyn) [inline EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj:13]: Right click on DataAccess Project, Unload Project, edit csproj file by removining this lines, no need in them.

**Review by tvardero — CHANGES_REQUESTED** [review]

**Review by tvardero — APPROVED** [review]

**Review by DimaDiD — APPROVED** [review]

File: EPlast/EPlast.WebApi/Controllers/CourseController.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            return Ok(result);
+        }
+
+        [HttpGet("{userId}")]
+        [Authorize(AuthenticationSchemes = "Bearer")]
+        public async Task<IActionResult> GetAllCourseByUserId(string userid)
+        {
+            var result = await _usercourseService.GetCourseByIdAsync(userid);
+            return Ok(result);
+        }
```
Comment (Serhii-Buniak) [inline EPlast/EPlast.WebApi/Controllers/CourseController.cs:35]: return BadResult if user don'n exist

**Review by Serhii-Buniak — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.WebApi/Controllers/CourseController.cs (hunk 2 above)
Comment (Goven112) [reply EPlast/EPlast.WebApi/Controllers/CourseController.cs:40]: done

Comment (tvardero) [reply EPlast/EPlast.WebApi/Controllers/CourseController.cs:40]: `GetCourseByIdAsync` returns empty IEnumerable, if not found.

**Review by tvardero — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.WebApi/Controllers/CourseController.cs (hunk 2 above)
Comment (Goven112) [reply EPlast/EPlast.WebApi/Controllers/CourseController.cs:65]: done

**Review by tvardero — APPROVED** [review]

**Review by Serhii-Buniak — APPROVED** [review]

**Pull request merged**